src/content/getting-started/observe-your-clusters-and-apps/_index.md

```diff
@@ -2,14 +2,55 @@
 title: Observe your clusters and apps
 description: Check cluster and app metrics and logs with the observability tools provided with the Giant Swarm platform.
 weight: 70
-last_review_date: 2024-04-05
+last_review_date: 2024-09-25
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-docs
 user_questions:
   - How do I observe the platform metrics and logs for my application?
   - What do I need to do to observe the platform metrics and logs for my application?
 ---
 
-<!--
-Show how to access Grafana and basic dashboard and logs
- -->
+
+<Explanation how monitoring works in two paragraphs>
```

Comment:
@QuentinBisson, it would be great if you are able to add an intro here; you can write it using the suggestion function. I take the rest of the page so I practice it

Reply:
I think this might cover it. @Rotfuks as you have better writing skills :D 

```suggestion
Each cluster created on the Giant Swarm platform benefits from our monitoring feature which allows us to provide you with 24/7 support to ensure best quality of service. The metrics scraped by a in-cluster agent and sent to a central location on the management cluster for you to be able to interact with them through our managed Grafana instance.
```